apps/frontend/src/app/3_organisms/ZeroLocForm/components/FormContent.tsx

```diff
@@ -401,6 +435,18 @@ export const FormContent: FC<FormContentProps> = props => {
           placeholder="0"
         />
       </FormGroup>
+
+      <AdvancedSettings
+        amount={props.maxOriginationFeeRate}
```

Comment:
Since `maxOriginationFeeRate` is initialized to `0`, you'll display `0` until a user changes it. This is incorrect (according to the spec: https://sovryn.atlassian.net/browse/SOV-1314?focusedCommentId=16267) and you should use `maxBorrowingFeeRate` as the initial value.